src/Microsoft.Azure.SignalR/HubHost/ServiceHubDispatcher.cs

```diff
@@ -76,15 +76,24 @@ public async Task ShutdownAsync()
                 return;
             }
 
-            using CancellationTokenSource source = new CancellationTokenSource();
-            source.CancelAfter(_options.GracefulShutdown.Timeout);
+            try
+            {
+                var source = new CancellationTokenSource(_options.GracefulShutdown.Timeout);
 
```

Comment:
how about adding an info log here to notify customer that we are entering graceful shutdown?